content/ADR-290-remove-profile-snapshots.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: adr
+adr: 290
+title: Remove Profile Snapshots
+date: 2025-11-10
+status: Draft
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - LautaroPetaccio
+  - pentreathm
+---
+
+## Abstract
+
+This ADR proposes the deprecation and eventual removal of the snapshots from profile entities. The change will be implemented in two phases: first, making the snapshot property and its related files optional for profile deployments, and second, rejecting any profile deployment that includes the snapshot property or any content files after a three-month transition period. This change is enabled by the profile-images service, which generates these images on-demand, eliminating the need for clients to upload them during profile deployment.
+
+## Context, Reach & Prioritization
+
+Profile entities currently require clients to upload snapshot images (`face256.png` and `body.png`) as part of the deployment process, as specified in [ADR-51](/adr/ADR-51) and [ADR-158](/adr/ADR-158). These validations ensure that profile deployments include these image files in their content.
+
+However, this approach has several drawbacks:
+
+1. **Storage inefficiency**: Requiring every profile to store snapshot images significantly increases storage requirements across all Catalyst nodes. With an unbounded amount of profiles deployed per player, this adds up to substantial storage costs.
```

Comment:
NIT:
Dangling content should be removed by the GC, which wouldn't be the issue, but the possibility of having infinite quantity of users in the platform

```suggestion
1. **Storage inefficiency**: Requiring every profile to store snapshot images significantly increases storage requirements across all Catalyst nodes. With an unbounded amount of profiles, this adds up to substantial storage costs.
```